src/main/java/com/google/impactdashboard/database_manager/data_update/DataUpdateManagerImpl.java

```diff
@@ -28,5 +44,32 @@ public void updateRecommendations(List<Recommendation> recommendations) {
    */
   @Override
   public void updateIAMBindings(List<IAMBindingDatabaseEntry> iamBindingsData) {
+    QueryJobConfiguration queryConfiguration = queryConfigurationBuilder
+      .insertValuesIAMTableConfiguration(iamBindingsData).build();
+    database.updateDatabase(queryConfiguration);
+  }
+
+  /**
+   * Deletes data from the IAM Bindings table that is over 365 days old. More 
+   * specifically, deletes data that is more than 365 older than the newest entry 
+   * in the table, since the tables should hold at most a window of 365 days 
+   * of information.
```

Comment:
I didn't realize this was the case when reviewing the original PR. 

This creates a dependency that the newest data must exist already in order to delete the old. Instead, could you alter the underlying query to use the current date - 365 to delete the old data?

Reply:
Good catch! I'll get rid of that dependency.